### PR TITLE
Finally add annismckenzie

### PR DESCRIPTION
**Pull request by annismckenzie**
I can't believe that I'm a contributor but don't have a profile. \*gasps\*

LGTM.

### CODE REVIEW

File: config/travellers/thegcat.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ hasbeen:
     # * wroc_love.rb
     - {"Wrocław": "Wrocław, Polska"}
     # * germany.rb
-    - {"Leipzig": "Leipzig, Detuschland"}
+    - {"Leipzig": "Leipzig, Deutschland"}
```
Comment (annismckenzie) [inline config/travellers/thegcat.yml]: Just a tiny correction.

**Comment by bascht** [comment]
HOW ON EARTH HAVE WE MISSED THIS?

Also: You've been to Wrocław? :D

**Comment by annismckenzie** [comment]
Have not! That's @thegcat's traveller file where I shamelessly corrected a spelling mistake.

**Comment by bascht** [comment]
You…

…did *what*?

**Comment by annismckenzie** [comment]
I GOT A GIF(T)! :dancers:

And I did. Had to appease my inner demon.

**Comment by thegcat** [comment]
Danke!

**Pull request merged**